helm/values-dev.yaml

```diff
@@ -2,6 +2,35 @@ microservice-chart:
   namespace: "ecommerce"
   nameOverride: ""
   fullnameOverride: ""
+  canaryDelivery:
+    create: true
+    ingress:
+      create: true
+      canary:
+        type: bluegreen
+    service:
+      create: true
+    deployment:
+      create: true
+      image:
+        repository: pagopadcommonacr.azurecr.io/pagopaecommerceeventdispatcherservice
+        tag: "latest"
+        pullPolicy: Always
+      envConfig:
+        TRANSACTIONS_CLOSE_PAYMENT_QUEUE_NAME: pagopa-d-weu-ecommerce-transactions-close-payment-queue-b
```

Comment:
those `blue` deployment queues are currently only created for [UAT env](https://github.com/pagopa/pagopa-infra/blob/c4ddf1de28227fac5308825439cff8a936a89aff/src/domains/ecommerce-common/03_storage.tf#L124), should we create those queues also for DEV env?